src/client/sandbox/node/element/index.js

```diff
@@ -36,6 +37,7 @@ export default class ElementSandbox extends SandboxBase {
         this.uploadSandbox = uploadSandbox;
         this.iframeSandbox = iframeSandbox;
         this.eventSandbox  = eventSandbox;
+        this.styleSandbox  = new StyleSandbox(this);
```

Comment:
Another sandboxes instantiated in `sandbox.js` and  attached in `sandbox.attach` method.
Do same for the `StyleSandbox`.

Reply:
NodeSandbox
https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/node/index.js#L34-L36